### DISCUSSION TITLE
[Feature Request]: Autohide feature to waybar

### DISCUSSION THREAD

**Discussion by shabbir23ah** [prasanthrangan/hyprdots#1013] [Ideas]
Can you please implement auto hide feature to waybar? It would be great.

I have seen the feature was released earlier here , https://github.com/Alexays/Waybar/pull/2248

**Comment by kRHYME7** [comment]
@shabbir23ah  Was the feature merge? The repo has been deleted. 

Let's try to check the manpage

**Comment by Enigma1309** [comment]
for now you can kill waybar  the default key is ctrl+esc in his keybinds config, it will switch between kill and alive state of waybar

**Comment by kRHYME7** [comment]
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/e8888ea2-1b3d-442b-8c5b-d2afe52ccbae)


@shabbir23ah  After reading the pull request and waybar's manpage I can't see if the feature is even implemented. 
Here's how I manage to "hide" waybar.  ``` bind = ALT_R, Control_R, exec, killall -SIGUSR1 waybar # Hide Waybar  ```. Waybar is capable to hide using ``` -SIGUSR1 ``` . (blur will still be present, this is a hyprland matter)

For now I'll label this as unrelated as this is out of scope of the repo. And I'll move this to discussion panel for future references.  
I suggest that you move/open this Feature request to the [waybar github page. 
](https://github.com/Alexays/Waybar/pull/2248)

⚡
